components/com_kunena/admin/install/kunena.install.upgrade.xml

```diff
@@ -641,6 +641,13 @@
 				AFTER `userid`
 			</query>
 		</version>
+		<version version="4.0.0-DEV"
+		         versiondate="2015-03-26"
```

Comment:
You need to set for vesion : 4.0.0-ALPHA2 else the row icon isn't added in table